src/ui/load.js

```diff
@@ -26,44 +30,45 @@ export async function load(): Promise<LoadResult> {
   // than ternaries. There's also a lot of repeated code here
 
   const diskStorage = createLedgerDiskStorage("data/ledger.json");
+  const networkStorage = new NetworkStorage("");
   const ledgerManager = new LedgerManager({
     storage: diskStorage,
   });
 
   const queries = [
-    fetch("sourcecred.json"),
-    fetch("static/server-info.json"),
-    fetch("config/currencyDetails.json"),
-    fetch("output/credGraph.json"),
+    loadJson(networkStorage, "sourcecred.json", (Combo.raw: any)),
+    loadJson(networkStorage, "static/server-info.json", (Combo.raw: any)),
+    loadJsonWithDefault(
+      networkStorage,
+      "config/currencyDetails.json",
+      currencyParser,
+      defaultCurrencyConfig
+    ),
+    loadJsonWithDefault(
+      networkStorage,
+      // TODO: Write Parser for CredGraph
+      "output/credGraph.json",
+      Combo.fmap(((Combo.raw: any): Combo.Parser<CredGraphJSON>), (graphJson) =>
+        CredGraph.fromJSON(graphJson)
+      ),
```

Comment:
why not use the parser from the CredGraph module? https://github.com/sourcecred/sourcecred/blob/master/src/core/credrank/credGraph.js#L128-L136